API.md

```diff
@@ -215,6 +289,17 @@ An `HTTP 404 Not Found` status code is returned if the case with the specified I
 
 An `HTTP 204 No Content` status code is returned if there are no cases found with the specified party ID.
 
+## Find Cases by Survey ID
+* `GET /cases/surveyid/{surveyId}` will return a list of cases with the given survey ID.
+
+### Example JSON Response
+```json
+[
+]
+```
+
+NOTE: For some reason, even when a valid survey ID is entered, the endpoint just returns a blank json message. Since no error message is generated, it can be assumed that the endpoint works, but is most likely redundant.
```

Comment:
We should looks for usages of this endpoint, if it is not used then we should remove both the endpoint and the documentation

Reply:
The purpose of this Trello card was to merely document the endpoints and mark any missing ones. Actually going into the code and deleting the endpoint is beyond the scope of the card, and should be dealt as part of a new card.

Reply:
Isn't documenting unused endpoints an exercise in futility. If we need to make a card for every application of the boy scout rule, then something's very wrong

Reply:
I understand your point, but I have been progressing through these API endpoint documentation   cards, making notes on unused endpoints in accordance with what the Trello card says ("Record redundant endpoints" is one of the tasks, not "delete redundant endpoints"). Whenever I have been submitting pull requests in the past for these endpoint documentation tasks, the rest of the group has approved of me merely documenting the unused endpoints, rather than deleting them. I believe this PR should be treated the same way.

If you disagree with this philosophy, I would recommend taking it up with some of the other members of the group, seeing as how they were the ones who instructed me to go about documenting the endpoints in this way.

Reply:
are we at least creating cards to track the endpoints that need to be removed?

Reply:
I'd agree that a card should at least be created to look at removing the unused endpoint (and any others which you may find).  It wouldn't feel right to find an issue and not have a follow on task to fix it. 👍

Reply:
I will create a card containing the list of endpoints that need to be deleted.